language/en-GB/tpl_cassiopeia.ini

```
@@ -4,6 +4,7 @@
 ; Note : All ini files need to be saved as UTF-8
 
 TPL_CASSIOPEIA_BACKTOTOP="Back to Top"
+TPL_CASSIOPEIA_BACKTOTOP_LABEL="Back-to-top Button"
```

technically its a link not a button


```suggestion
TPL_CASSIOPEIA_BACKTOTOP_LABEL="Back-to-top Link"
```